src/output/time.rs

```diff
@@ -113,6 +120,21 @@ fn long_zoned(time: SystemTime, zone: &TimeZone) -> String {
             date.hour(), date.minute())
 }
 
+#[allow(trivial_numeric_casts)]
+fn relative(time: SystemTime) -> String {
+    format!(
+        "{:>13}", 
```

Comment:
This seems like an arbitrary value. instead of relying what seems like a working column size, it would make more sense to not pad in here but just return the formatted relative time and afterwards collect the maximum length to pad to that afterwards.

Reply:
after a little more digging in the codebase I found the correct place to right-align columns and change this.